Assets/Scripts/PlayerSpawnTrigger.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerSpawnTrigger : MonoBehaviour
+{
+	private Transform playerRespawnTransform;
+	// Start is called before the first frame update
+	void Start()
+	{
+		playerRespawnTransform = GameObject.Find("Game Manager").transform.Find("PlayerRespawnPoint").transform;
```

Comment:
Does this `transform.Find("PlayerRespawnPoint")` mean the initial spawn point of the player? If yes, maybe you could rename it to just "PlayerSpawnPoint", because now it's a bit misleading?

Reply:
no, its the one where you spawn after dying

Reply:
But is it the same as the initial spawn point? So `playerRespawnTransform` at the beginning is set to the same place or is it different?

Reply:
It is the initial spawn point too, we just change its position once checkpoints are triggered.